Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: harbor
-version: 1.4.0-dev
+version: 1.4.1-dev
```

Comment:
nit: you don't need to update it.  The maintainer will edit it when the release branch is cut.